src/v/kafka/server/fetch_metadata_cache.h

```diff
@@ -54,6 +57,19 @@ class fetch_metadata_cache {
         auto& e = _cache[ktp];
         e.reset_ts();
         if (offset_count > 0) {
+            if (e.bytes_per_offset.has_samples()) {
+                auto avg_bytes_per_offset = static_cast<double>(
+                  e.bytes_per_offset.get());
+                auto bytes_per_offset = static_cast<double>(total_size_bytes)
+                                        / static_cast<double>(offset_count);
+                auto abs_err = std::max(bytes_per_offset, avg_bytes_per_offset)
+                               - std::min(
+                                 bytes_per_offset, avg_bytes_per_offset);
+                const auto epsilon = 1e-6; // prevent division by zero
+                auto emr = abs_err / (avg_bytes_per_offset + epsilon);
```

Comment:
Don't you have to divide by `bytes_per_offset` here as that's the "true" number?